src/useDownload.tsx

```diff
@@ -53,6 +54,8 @@ export function useDownload() {
       (tabs) => {
         const tabId = tabs[0].id || 0;
         setTabId(tabId);
+		const url = tabs[0].url?.endsWith('documentArchiveListFormAction.do') ? tabs[0].url : 'https://konto.flatex.at/banking-flatex.at/documentArchiveListFormAction.do' ;
```

Comment:
I would prefer to parse the hostname via the `URL` constructor and then pass the hostname to the DOMEvaluator content script. So differentiating between DE/AT would be cleaner in the future.


```suggestion
    // something something like
		const url = new URL(tabs[0].url);
    const { hostname } = url // konto.flatex.at | konto.flatex.de
```